specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,1430 @@
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskResultsKind"
+        }
+      }
+    },
+    "CustomConversationalTask": {
+      "description": "The input for a custom conversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTask"
+        }
+      ],
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/ConversationAnalysisOptions"
+        },
+        "parameters": {
+          "$ref": "#/definitions/CustomConversationTaskParameters"
+        }
+      },
+      "x-ms-discriminator-value": "CustomConversation",
+      "required": [
+        "analysisInput",
+        "parameters"
+      ]
+    },
+    "CustomConversationTaskParameters": {
+      "type": "object",
+      "description": "Input parameters necessary for a Custom Conversation Task.",
+      "properties": {
+        "projectName": {
+          "type": "string",
+          "description": "The name of the project to use."
+        },
+        "deploymentName": {
+          "type": "string",
+          "description": "The name of the specific deployment of the project to use."
+        },
+        "verbose": {
+          "type": "boolean",
+          "description": "If true, the service will return more detailed information in the response."
+        }
+      },
+      "required": [
+        "projectName",
+        "deploymentName"
+      ]
+    },
+    "CustomConversationalTaskResult": {
+      "description": "The results of a custom conversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTaskResult"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "results": {
+          "$ref": "#/definitions/AnalyzeConversationResult"
+        }
+      },
+      "required": [
+        "results"
+      ],
+      "x-ms-discriminator-value": "CustomConversationResult"
+    },
+    "ConversationItemBase": {
+      "type": "object",
+      "discriminator": "modality",
+      "description": "The abstract base for a user input formatted conversation (e.g., Text, Transcript).",
+      "properties": {
+        "participantId": {
+          "description": "The user input unique identifier corresponding each participant in the conversation",
+          "type": "string"
+        },
+        "id": {
+          "description": "The user input unique identifier corresponding the conversation Item (utterance)",
+          "type": "string"
+        },
+        "language": {
+          "description": "The input language of the conversation item",
+          "type": "string"
+        },
+        "modality": {
+          "description": "The modality (format) of the conversation item (e.g., Text, Transcript)",
+          "type": "string",
+          "enum": [
+            "text"
+          ],
+          "x-ms-enum": {
+            "name": "modality",
+            "modelAsString": false
```

Comment:
So `ConversationItemBase` will *never* be used for output? If there's any doubt, make this `true` or some SDKs will have future problems.

Reply:
Should be true. Updated.